re/postgres/init.go

```diff
@@ -50,6 +50,33 @@ func Migration() *migrate.MemoryMigrationSource {
 					`ALTER TABLE rules DROP COLUMN tags;`,
 				},
 			},
+			{
+				Id: "rules_03",
+				Up: []string{
+					`CREATE TABLE IF NOT EXISTS rules_logs (
+						id            VARCHAR(36) PRIMARY KEY,
+						rule_id       VARCHAR(36) NOT NULL,
+						rule_name     VARCHAR(1024) NOT NULL,
+						domain_id     VARCHAR(36) NOT NULL,
```

Comment:
This is probably not needed, since rule already has them. If needed, we can join by id.